packages/frontend/src/commons/index.ts

```diff
@@ -1,43 +1,26 @@
 import { ChainId } from "@carrot-kpi/sdk";
 import { SelectOption } from "@carrot-kpi/ui";
-import { FunctionComponent, SVGProps } from "react";
-import { Address } from "wagmi";
 import { ReactComponent as CarrotIcon } from "../assets/carrot.svg";
+import { OptionWithIcon } from "../creation-form/types";
 
 export type SupportedChain = Extract<ChainId, ChainId.SEPOLIA | ChainId.GOERLI>;
 export const SupportedChain = {
     [ChainId.SEPOLIA]: ChainId.SEPOLIA,
     [ChainId.GOERLI]: ChainId.GOERLI,
 } as const;
 
-export interface Arbitrator {
-    address: Address;
-    name: string;
-    icon: FunctionComponent<SVGProps<SVGSVGElement>>;
-}
-
-export const EPLORER_BY_CHAIN: Record<SupportedChain, string> = {
```

Comment:
We don't need these mappings anymore?

Reply:
Nono the explorer is available using `wagmi`, and the Reality.eth address is returned by the oracle contract.